src/Datadog.Trace.ClrProfiler.Managed/Instrumentation.cs

```diff
@@ -38,7 +38,7 @@ public static bool ProfilerAttached
                 {
                     return NativeMethods.IsProfilerAttached();
                 }
-                catch (DllNotFoundException)
+                catch
```

Comment:
This is a behavior change. With this change we now return `false` which indicates that the profiler is _not_ attached, even if the profiler _is_ attached and an exception was thrown for a different reason.

Since `ProfilerAttached` is only used in our test apps, should we allow exception to bubble up instead of hiding them?

Reply:
The exception that's thrown when we don't have permission to call native code is a `SecurityException`. I think you're right, maybe we should just let that bubble up to the caller. I will revert this

Reply:
Resolved in cbf5a666e4df5b16ba2009dde35bbb535a8230d3